pkg/boot/uefi/uefi.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package uefi
+
+import (
+	"bytes"
+	"debug/pe"
+	"encoding/binary"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"math"
+	"os"
+
+	"github.com/u-root/u-root/pkg/acpi"
+	"github.com/u-root/u-root/pkg/boot"
+	"github.com/u-root/u-root/pkg/boot/kexec"
+	"github.com/u-root/u-root/pkg/smbios"
+)
+
+const fvEntryImageOffset int64 = 0xA0
+
+var bootExecute = boot.Execute
+var kexecLoad = kexec.Load
+var kexecParseMemoryMap = kexec.ParseMemoryMap
+var getRSDP = acpi.GetRSDP
+
+func getSMBIOSBaseInternal() (int64, int64, error) {
```

Comment:
this could be a func in the smbios package

Reply:
Done.